docs/reference/cluster/stats.asciidoc

```diff
@@ -38,6 +38,278 @@ include::{docdir}/rest-api/common-parms.asciidoc[tag=flat-settings]
+Total count of hits, misses, and cached queries in the query cache.
+
+`indices.query_cache.hit_count`::
+(integer)
+Number of query cache hits.
+
+`indices.query_cache.miss_count`::
+(integer)
+Number of query cache misses.
+
+`indices.query_cache.cache_size`::
+(integer)
+Size, in bytes, of the query cache.
+
+`indices.query_cache.cache_count`::
+(integer)
+Count of queries in the query cache.
+
+`indices.query_cache.evictions`::
+(integer)
+Number of query cache evictions.
+
+`indices.completion.size_in_bytes`::
+(<<byte-units, byte units>>)
+Memory used for completion.
+
+`indices.completion.size_in_bytes`::
+(integer)
+Memory, in bytes, used for completion.
+
+`indices.segments.count`::
+(integer)
+Number of segments.
+
+`indices.segments.memory`::
+(<<byte-units, byte units>>)
+Size of segments.
+
+`indices.segments.memory_in_bytes`::
+(integer)
+Size, in bytes, of segments.
+
+`indices.segments.terms_memory`::
+(<<byte-units, byte units>>)
+Size of terms in segments.
+
+`indices.segments.terms_memory_in_bytes`::
+(integer)
+Size, in bytes, of terms in segments.
+
+`indices.segments.stored_fields_memory`::
+(<<byte-units, byte units>>)
+Size of stored fields in segments.
+
+`indices.segments.stored_fields_memory_in_bytes`::
+(integer)
+Size, in bytes, of stored fields in segments.
+
+`indices.segments.term_vectors_memory`::
+(<<byte-units, byte units>>)
+Size of term vectors in segments.
+
+`indices.segments.term_vectors_memory_in_bytes`::
+(integer)
+Size, in bytes, of term vectors in segments.
+
+`indices.segments.norms_memory`::
+(<<byte-units, byte units>>)
+Size of normalization factors in segments.
+
+`indices.segments.norms_memory_in_bytes`::
+(integer)
+Size, in bytes, of normalization factors in segments.
+
+`indices.segments.points_memory`::
+(<<byte-units, byte units>>)
+Size of points in segments.
+
+`indices.segments.points_memory_in_bytes`::
+(integer)
+Size, in bytes, of points in segments.
+
+`indices.segments.doc_values_memory`::
+(<<byte-units, byte units>>)
+Size of doc values in segments.
+
+`indices.segments.doc_values_memory_in_bytes`::
+(integer)
+Size, in bytes, of doc values in segments.
+
+`indices.segments.index_writer_memory_in_bytes`::
+(<<byte-units, byte units>>)
+Memory used by the index writer.
+
+`indices.segments.index_writer_memory_in_bytes`::
+(integer)
+Memory, in bytes, used by the index writer.
+
+`indices.segments.version_map_memory`::
+(<<byte-units, byte units>>)
+Memory used by the version map.
+
+`indices.segments.version_map_memory_in_bytes`::
+(integer)
+Memory, in bytes, used by the version map.
+
+`indices.segments.fixed_bit_set_memory`::
+(<<byte-units, byte units>>)
+Memory used by fixed bit sets for nested object field types and type
+filters for <<parent-join,join>> fields.
+
+`indices.segments.fixed_bit_set_memory_in_bytes`::
+(integer)
+Memory, in bytes, used by fixed bit sets.
+
+`indices.segments.max_unsafe_auto_id_timestamp`::
+(integer)
+Timestamp of the most recent retry request.
```

Comment:
Timestamp of the most recently retried indexing request.

Reply:
Fixed with 5d8796747b2.